### PR TITLE
Add VideoSwinTransformer

### PR DESCRIPTION
**Pull request by HydrogenSulfate**
1. Release VideoSwin on K400 dataset
2. Update some code.
3. Raname all `seg_num` to `num_seg`
4. Add runtime config options `runtime_cfg` in transformer based yamls, in order to separate the variables used by the model from the backbone, make it easier to control the behavior of the model at runtime
- [x] add pretrained 3D ckpt
- [x] add pretrained 2D ckpt
- [x] add inference path
- [x] update README
- [x] rename pretrained name in pptimesformer.

### CODE REVIEW

File: docs/zh-CN/tutorials/accelerate.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ otl = []
 input = paddle.randn(SHAPE)
 for i in range(10000):
     t1 = time.time()
-    out1 = F.temporal_shift(x=input, seg_num=2, shift_ratio=0.2)
+    out1 = F.temporal_shift(x=input, num_seg=2, shift_ratio=0.2)
```
Comment (huangjun12) [inline docs/zh-CN/tutorials/accelerate.md]: api接口没改

File: main.py [hunk 2]
```diff
@@ -88,9 +92,10 @@ def main():
     if seed is not None:
         assert isinstance(
             seed, int), f"seed must be a integer when specified, but got {seed}"
-        paddle.seed(seed)
-        np.random.seed(seed)
         random.seed(seed)
+        np.random.seed(seed)
+        paddle.seed(seed)
+        paddle.framework.seed(seed)
```
Comment (huangjun12) [inline main.py]: 这个和paddle.seed的区别是？

File: docs/zh-CN/tutorials/accelerate.md (hunk 1 above)
Comment (HydrogenSulfate) [reply docs/zh-CN/tutorials/accelerate.md]: done

File: main.py (hunk 2 above)
Comment (HydrogenSulfate) [reply main.py]: 应该是一样的，`paddle.seed`是从framework里import过来的

**Comment by huangjun12** [comment]
LSTM

**Pull request merged**